### DISCUSSION TITLE
[A-L] (2023/24) Foglio 1 - Esercizio 5

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#136] [Esercizi - Bertini]
![Es 5](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/0bd60db1-c1f0-4487-b1d3-1ed6d37ab9bc)

**Comment by Elia-Belli** [comment]
Sia $I$ l'insieme degli intervistati $|I| = 60$,
sia $To$ l'insieme delle persone che leggono Topolino $|To|=25$
sia $Te$ l'insieme delle persone che leggono Tex $|Te|=26$
sia $D$ l'insieme delle persone che leggono Diabolik $|D|=23$
sappiamo che $|To \cap Te|=9$ ; $|To \cap D|= 11$ ; $|Te \cap D|=8$ ; $|To \cap Te \cap D|=3$ 

- **1)** Le persone che leggono solo Topolino sono: $|To| - |To \cap D| - |To \cap Te| + |To \cap Te \cap D|= 25 - 11 - 9+3 = 8$
- **2)** Le persone che leggono solo Tex sono: $|Te| - |Te \cap D| - |Te \cap To| +|To \cap Te \cap D|= 26-8-9 +3= 12$
- **3)** Le persone che leggono solo Diabolik sono: $|D| - |To \cap D| - |D \cap Te|+|To \cap Te \cap D| = 23-11-8+3= 7$
- **4)** Le persone che leggono almeno un periodico ("che leggono") sono:
$|To \cup Te \cup D| = |To| + |Te| + |D| -|To \cap Te|-|To \cap D| - |Te \cap D| + |To \cap Te \cap D| $ 
$= 25+26+23 -9-11-8+3 = 49 $
- **5)** Le persone che leggono solo uno dei tre periodici sono: $8+12+7 = 27$
- **6)** Le persone che non leggono nulla sono il totale degli intervistati meno le persone che leggono: $60 - 49 = 11$

**Comment by CiottoloMaggico** [comment]
@Elia-Belli nei primi 3 punti hai rimosso 2 volte il numero di persone che leggono tutti i fumetti.
Prendendo ad esempio il primo punto,
| Topolino ∩ Diabolic | e | Topolino ∩ Tex | contengono entrambi il gruppo di persone che legge tutti e 3 i fumetti.
Il risultato corretto dovrebbe essere: |Topolino| - |Topolino ∩ Diabolik| - |Topolino ∩ Tex| + |Topolino ∩ Diabolik ∩ Tex|
Di conseguenza anche il punto 5 dovrebbe essere ricalcolato
